internal/service/annotations/ingress.go

```diff
@@ -14,21 +15,25 @@ const (
 )
 
 // FillLBWithIngressAnnotations prepares the LB input based on annotations.
-func FillLBWithIngressAnnotations(lbInput *serverscom.L7LoadBalancerCreateInput, annotations map[string]string) *serverscom.L7LoadBalancerCreateInput {
+func FillLBWithIngressAnnotations(client *serverscom.Client, lbInput *serverscom.L7LoadBalancerCreateInput, annotations map[string]string) (*serverscom.L7LoadBalancerCreateInput, error) {
 	// LBStoreLogsRegionCode annotation
 	if value, ok := annotations[LBStoreLogsRegionCode]; ok {
-		val, err := strconv.Atoi(value)
-		if err == nil {
-			lbInput.StoreLogsRegionID = &val
+		regionID, err := GetRegionIDByCode(context.Background(), client.CloudComputingRegions, value)
```

Comment:
This should not be a cloud computing region. This should be a cloud storage region, but it seems we don't have it in our API right now.